packages/beacon-node/src/chain/seenCache/seenGossipBlockInput.ts

```diff
@@ -310,6 +310,7 @@ export class SeenGossipBlockInput {
           };
           resolveAvailability(blockData);
           metrics?.syncUnknownBlock.resolveAvailabilitySource.inc({source: BlockInputAvailabilitySource.GOSSIP});
+          metrics?.dataColumns.bySource.inc({source: blockData.dataColumnsSource}, blockData.dataColumns.length);
```

Comment:
We dont need to do a collection here.  This is for blocks only (when there are no `blobKzgCommitments` in the block)